src/components/Category.js

```diff
@@ -0,0 +1,11 @@
+import './styles/Category.css';
+
+const Category = () => (
+  <div className="categories">
+    <button type="button" className="status-btn">
+      Coming Soon
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md) for this project, " `Categories:
Should display a button "Check status" only.` ", please kindly fix that.